Corrected RTL issues new forum post button

OmarIthawi opened the pull request:
- Word break in Arabic on Google Chrome
   - Refactored forum buttons


This is a bug we found in our fork (https://github.com/Edraak/edx-platform/pull/180), so I thought to open a PR for it here.

**Before:**

- ~~The icon is floated to left, like the LTR. It should be flipped.~~ The icon will be removed in #12538 so no need to fix it now.
- The [text is broken into two lines in Google Chrome](http://stackoverflow.com/questions/29078128/why-do-arabic-text-breaks-into-two-lines-sometimes).

![screen shot 2016-06-01 at 5 54 16 pm](https://cloud.githubusercontent.com/assets/645156/15714948/4f54c8a4-2825-11e6-8713-a2e44753825e.png)


**After:**

- All is fine

![screen shot 2016-06-01 at 5 54 26 pm](https://cloud.githubusercontent.com/assets/645156/15714959/590fe978-2825-11e6-8c33-186580a67f4d.png)

edx-webhook commented:
Thanks for the pull request, @OmarIthawi! I've created [OSPR-1274](https://openedx.atlassian.net/browse/OSPR-1274) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

andy-armstrong commented:
Thanks for the PR @OmarIthawi. This is good timing as I've also been working on the UX for the discussion forums in this PR:

https://github.com/edx/edx-platform/pull/12538

In particular, we have now removed the icon, so the only issue remaining will be word wrapping in button labels. I wonder whether this should be done in a more global location, as buttons will rarely look good when wrapped. I'll have a dig around in the code and see what I can find.

andy-armstrong commented:
BTW, I was suffering from the same three Bok Choy failures. I'm surprised that your PR would have this issue as you haven't changed anything that should affect the login page. We should be able to work around it in your PR once mine lands.

@benpatterson FYI, here's another instance of the screenshot failure.

OmarIthawi commented:
@andy-armstrong I've reviewed the discussion code and found that it is isolated from the other code i.e. the buttons have their own CSS.

I think this is good for now.

OmarIthawi commented:
Regarding the icon, I reverted the fix.

OmarIthawi commented:
@andy-armstrong in order to make this PR mergable, I've already removed the icon fix, and rebased the branch to refresh the automated checks status hopefully the bokc-hoy issues gets fixes.

OmarIthawi commented:
jenkins run lettuce

andy-armstrong commented:
jenkins run lettuce

andy-armstrong commented:
👍  this looks great, so I'll merge it now. Thanks for the contribution
